tests/ledctl/ledctl_cmd.py

```diff
@@ -52,11 +52,30 @@ def run_ledctl_cmd(self, params: list, output=False, check=True):
         params.insert(1, self.bin)
 
         LOGGER.debug(f"Command: {params}")
-        return subprocess.run(params, capture_output=output, check=check)
+        print(params)
```

Comment:
this cannot be accepted in upstream version, you should use logger

Reply:
Right, that was just for debug, I missed it.